src/jquery.uls.core.js

```diff
@@ -354,7 +356,6 @@
 		},
 
 		click: function ( e ) {
-			e.stopPropagation();
```

Comment:
I did https://github.com/wikimedia/jquery.uls/pull/202 without looking at this. Sorry, this patch was not in my memory.

I think we can further simplify this by not doing anything with events.